Book-A-Desk.Api/EmailNotification.fs

```diff
@@ -0,0 +1,46 @@
+namespace Book_A_Desk.Api
+
+open System
+open System.Net.Mail
+open Book_A_Desk.Api.Models
+open Book_A_Desk.Domain.QueriesHandler
+open Book_A_Desk.Domain.Office.Domain
+
+type EmailNotification =
```

Comment:
You can see this type as the equivalent of an interface in OOP. What you usually want is for this type to have a generic name. This would allow us in the future to do more than send an email without changing the interface.

A suggestion for this name:
``` fsharp
type BookingNotifier =
{
    Notify: Booking -> unit
}
```

Or, if you want:
``` fsharp
type BookingNotifier =
{
    NotifySuccess: Booking -> unit
}
```
Which would allow for a NotifyFailure to be called, or something like that.

Reply:
To Do: Refactor and rename to BookingNotifier.